R/FilterState-utils.R

```diff
@@ -265,7 +265,7 @@ init_filter_state.POSIXlt <- function(x,
 #' @keywords internal
 #'
 #' @examples
-#' \dontrun{
+#' \donttest{
```

Comment:
we want to run all examples during R CMD check and know which examples fail
```suggestion
#' \donttest{
```